test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -185,6 +185,22 @@ def forward(self, x):
         x = torch.randn(3, 4)
         self.run_test(MyModel(), x)
 
+    def test_scalar_tensor(self):
+        class test(torch.nn.Module):
+            def forward(self, input):
+                return torch.scalar_tensor(input.size(0)), \
+                    torch.scalar_tensor(input.size(1), dtype=torch.int64)
+
+        x = torch.randn(2, 3, 4)
+        y = torch.randn(7, 8, 9)
+        model = test()
+        self.run_test(model, x, test_with_inputs=[y],
+                      input_names=['input_1'],
+                      output_names=['output_1', 'output_2'],
+                      dynamic_axes={'input_1': [0, 1, 2],
+                                    'output_1': [0],
```

Comment:
For outputs they don't need to be dynamic in shape right? Are both the outputs here rank 0 tensor (scalar)?